README.md

```diff
@@ -1,16 +1,35 @@
 # Practical Tools for Simple Design
 
+###### Officially Supported Platforms and IDEs
+
+|                                | Windows | macOS | Linux |
+|:------------------------------:|:-------:|:-----:|:-----:|
+|             CLion              |    V    |   V   |   V   |
+|        VSCode[^codeoss]        |    V    |   V   |   V   |
+| Visual Studio[^novs][^vsmacos] |    ?    |   X   |   X   |
+|             No IDE             |    V    |   V   |   V   |
+
+[^vsmacos]: [Microsoft Kills Visual Studio for Mac](https://visualstudiomagazine.com/articles/2023/08/30/vs-for-mac-retirement.aspx)
```

Comment:
footnote not good, please put closely.

Reply:
![Hmm](https://cdn.7tv.app/emote/60ae3e54259ac5a73e56a426/1x.webp)